runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/External.java

```diff
@@ -78,7 +78,7 @@
   private static AtomicInteger namespaceCounter = new AtomicInteger(0);
 
   private static final ExpansionServiceClientFactory DEFAULT =
-      new DefaultExpansionServiceClientFactory(
+     DefaultExpansionServiceClientFactory.create(
```

Comment:
@iemejia looks like this indentation change breaks the build:

    Execution failed for task ':runners:core-construction-java:spotlessJavaCheck'.
    > The following files had format violations:
      runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/External.java
          @@ -78,7 +78,7 @@
           ··private·static·AtomicInteger·namespaceCounter·=·new·AtomicInteger(0);

           ··private·static·final·ExpansionServiceClientFactory·DEFAULT·=
          -·····DefaultExpansionServiceClientFactory.create(
          +······DefaultExpansionServiceClientFactory.create(
           ··········endPoint·->·ManagedChannelBuilder.forTarget(endPoint.getUrl()).usePlaintext().build());

           ··private·static·int·getFreshNamespaceIndex()·{
    Run './gradlew :runners:core-construction-java:spotlessApply' to fix these violations.

Reply:
Thanks for pointing this out! I did a quick fix now on master to unlock other people's PRs,

Reply:
This still seems to be broken, I put up https://github.com/apache/beam/pull/13445 for it.